### PR TITLE
Feature/mocking all requests from a single date

### PR DESCRIPTION
**Pull request by JJGothard**
Added a section on how to create a mock server for all of the requests made on a specific date. 

I tried to write this in a way that mirrored the rest of the doc as much as possible.

Thanks for reviewing!

### CODE REVIEW

File: src/pages/docs/postman/mock-servers/setting-up-mock.md [hunk 1]
```diff
@@ -128,7 +128,13 @@ If you already have mock servers created from the same collection, you will see
 
 ![Create new mock server from collection sidebar when there are existing mock servers](https://assets.postman.com/postman-docs/mock-cnx-collection-sidebar-existing.png)
 
-### Mocking a request from History
+### Mocking from History
```
Comment (SueSmith) [inline src/pages/docs/postman/mock-servers/setting-up-mock.md]: the toc link at the top of the page will need updated to match this

File: src/pages/docs/postman/mock-servers/setting-up-mock.md [hunk 2]
```diff
@@ -140,6 +146,32 @@ This process of creating a mock server will also create a collection with the sa
...
 
+#### Mocking all requests made on a specific date
+
+You can create a mock server from **History** for all the requests you made on a specific date, creating a new collection at the same time.
+
+In the Postman app, click **History**.
+
+![History tab](https://assets.postman.com/postman-docs/History+Mock+creation.jpg)
+
+Next to the date you want to mock all the requests for, click the ellipsis **(...)** and then click **Mock Requests**. The **Set up the mock server** modal will open.
```
Comment (SueSmith) [inline src/pages/docs/postman/mock-servers/setting-up-mock.md]: I think I'd move the last sentence here to after the next screengrab so it's next to the instructions for that modal

File: src/pages/docs/postman/mock-servers/setting-up-mock.md [hunk 3]
```diff
@@ -140,6 +146,32 @@ This process of creating a mock server will also create a collection with the sa
...
+
+![Ellipsis and mock requests](https://assets.postman.com/postman-docs/Mock+all+requests.jpg)
+
+In the modal, give your server a name. You also have the option to connect an environment to the server, choose if the server will be private or public, and to save the server's URL as an environment variable. Then click **Create mock server**.
```
Comment (SueSmith) [inline src/pages/docs/postman/mock-servers/setting-up-mock.md]: finding the second sentence a wee bit confusing, could we reword "choose if the server will be private or public" maybe just match the ui "make the server private"?

File: src/pages/docs/postman/mock-servers/setting-up-mock.md [hunk 4]
```diff
@@ -140,6 +146,32 @@ This process of creating a mock server will also create a collection with the sa
...
+
+![Mocks modal](https://assets.postman.com/postman-docs/Mocks+modal.jpg)
+
+> If you choose to make your mock server private, you'll have to generate and API key.
```
Comment (SueSmith) [inline src/pages/docs/postman/mock-servers/setting-up-mock.md]: think we have a typo here "and" > "an"

File: src/pages/docs/postman/mock-servers/setting-up-mock.md [hunk 5]
```diff
@@ -140,6 +146,32 @@ This process of creating a mock server will also create a collection with the sa
...
+
+Make note of the "Next steps" that will automatically appear after you click **Create mock server**.
```
Comment (SueSmith) [inline src/pages/docs/postman/mock-servers/setting-up-mock.md]: maybe be a bit more specific than "make note" here? main thing is they need the url..

File: src/pages/docs/postman/mock-servers/setting-up-mock.md [hunk 6]
```diff
@@ -140,6 +146,32 @@ This process of creating a mock server will also create a collection with the sa
...
+
+![Mocks modal](https://assets.postman.com/postman-docs/Next+steps+modal+RS.jpg)
+
+Creating the mock server will also create a collection with the same name as the server. There will be copies of all of the requests made on the date that you created the server from in your **History**. The server is connected to this new collection.
```
Comment (SueSmith) [inline src/pages/docs/postman/mock-servers/setting-up-mock.md]: for second sentence maybe "the collection will contain" instead of "there will be"

can we be a bit more specific for the last sentence, e.g. the requests are to the mock server url aren't they?

File: src/pages/docs/postman/mock-servers/setting-up-mock.md [hunk 7]
```diff
@@ -140,6 +146,32 @@ This process of creating a mock server will also create a collection with the sa
...
+
+![Mocks modal](https://assets.postman.com/postman-docs/New+server+and+collection+CS.jpg)
```
Comment (SueSmith) [inline src/pages/docs/postman/mock-servers/setting-up-mock.md]: I'd change the arrow and callout overlay graphics on this, they're a bit confusing, even removing them completely would be preferable imo 😬

File: src/pages/docs/postman/mock-servers/setting-up-mock.md (hunk 1 above)
Comment (JJGothard) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md:131]: Done

File: src/pages/docs/postman/mock-servers/setting-up-mock.md (hunk 2 above)
Comment (JJGothard) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md]: DOne

File: src/pages/docs/postman/mock-servers/setting-up-mock.md (hunk 3 above)
Comment (JJGothard) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md]: Done

File: src/pages/docs/postman/mock-servers/setting-up-mock.md (hunk 4 above)
Comment (JJGothard) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md]: DOne

File: src/pages/docs/postman/mock-servers/setting-up-mock.md (hunk 5 above)
Comment (JJGothard) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md]: Done

File: src/pages/docs/postman/mock-servers/setting-up-mock.md (hunk 7 above)
Comment (JJGothard) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md:175]: I used them only to be in keeping with the rest of the doc. 
Done.

**Comment by JJGothard** [comment]
@SueSmith , all comments have been addressed. Thanks!

File: src/pages/docs/postman/mock-servers/setting-up-mock.md [hunk 8]
```diff
@@ -140,6 +146,34 @@ This process of creating a mock server will also create a collection with the sa
...
+
+The **Set up the mock server** modal will open.
+
+In the modal, give your server a name. You also have the option to connect an environment to the server, make the sever private, and to save the server's URL as an environment variable. Then click **Create mock server**.
+
+![Mocks modal](https://assets.postman.com/postman-docs/Mocks+modal.jpg)
+
+> If you choose to make your mock server private, you'll have to generate an API key.
+
+"Next steps" will automatically appear. Read through them, and be sure to click the mock server's URL so that you can add examples to the server. Learn more about [examples and servers](/docs/postman/collections/examples/).
```
Comment (SueSmith) [inline src/pages/docs/postman/mock-servers/setting-up-mock.md:169]: I think I'd reword this paragraph to something like "You will see an overview of your new mock server details. Note the URL as you will need it to make requests that return mock data. The examples associated with each request will determine what data the mock server returns."

File: src/pages/docs/postman/mock-servers/setting-up-mock.md [hunk 9]
```diff
@@ -140,6 +146,34 @@ This process of creating a mock server will also create a collection with the sa
...
+
+![Mocks modal](https://assets.postman.com/postman-docs/Next+steps+modal+RS.jpg)
+
+Creating the mock server will also create a collection with the same name as the server. The collection will have copies of all of the requests made on the date that you created the server from in your **History**. All requets from this collection will be made to the server you just created.
```
Comment (SueSmith) [inline src/pages/docs/postman/mock-servers/setting-up-mock.md:173]: Still finding the second sentence a wee bit confusing, maybe something like "The collection will contain a copy of each request made on the date you chose in your __History__."

File: src/pages/docs/postman/mock-servers/setting-up-mock.md (hunk 8 above)
Comment (JJGothard) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md:169]: Done

File: src/pages/docs/postman/mock-servers/setting-up-mock.md [hunk 10]
```diff
@@ -46,7 +46,7 @@ Mocks in Postman are tied to a collection. Postman matches requests and generate
     * [Mocking from the Launchpad](#mocking-from-the-launchpad)
     * [Mocking from an API](#mocking-from-an-api)
     * [Mocking from the Collections sidebar](#mocking-from-the-collections-sidebar)
-    * [Mocking a request from History](#mocking-a-request-from-history)
+    * [Mocking from History](#mocking-from-history)
```
Comment (claireriley) [inline src/pages/docs/postman/mock-servers/setting-up-mock.md:49]: I'd add the sections under History so this ToC shows all headers on the page

File: src/pages/docs/postman/mock-servers/setting-up-mock.md [hunk 11]
```diff
@@ -140,6 +146,34 @@ This process of creating a mock server will also create a collection with the sa
 
 ![New collection created when creating mock server from history request](https://assets.postman.com/postman-docs/mock-cnx-history-request-created.png)
```
Comment (claireriley) [inline src/pages/docs/postman/mock-servers/setting-up-mock.md:147]: Another quick image update

File: src/pages/docs/postman/mock-servers/setting-up-mock.md [hunk 12]
```diff
@@ -140,6 +146,34 @@ This process of creating a mock server will also create a collection with the sa
...
+
+You can create a mock server from **History** for all the requests you made on a specific date, creating a new collection at the same time.
+
+In the Postman app, click **History**.
+
+![History tab](https://assets.postman.com/postman-docs/History+Mock+creation.jpg)
+
+Next to the date you want to mock all the requests for, click the ellipsis **(...)** and then click **Mock Requests**.
+
+![Ellipsis and mock requests](https://assets.postman.com/postman-docs/Mock+all+requests.jpg)
```
Comment (claireriley) [inline src/pages/docs/postman/mock-servers/setting-up-mock.md:159]: I don't think the arrows or numbers are necessary for any images here—can we remove all of them?

File: src/pages/docs/postman/mock-servers/setting-up-mock.md [hunk 13]
```diff
@@ -140,6 +146,34 @@ This process of creating a mock server will also create a collection with the sa
...
+
+The **Set up the mock server** modal will open.
+
+In the modal, give your server a name. You also have the option to connect an environment to the server, make the sever private, and to save the server's URL as an environment variable. Then click **Create mock server**.
```
Comment (claireriley) [inline src/pages/docs/postman/mock-servers/setting-up-mock.md:163]: "sever"

Comment (claireriley) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md:163]: I'd remove "then" in the last sentence

File: src/pages/docs/postman/mock-servers/setting-up-mock.md [hunk 14]
```diff
@@ -140,6 +146,34 @@ This process of creating a mock server will also create a collection with the sa
...
+
+![Mocks modal](https://assets.postman.com/postman-docs/Mocks+modal.jpg)
+
+> If you choose to make your mock server private, you'll have to generate an API key.
+
+The "Next steps" modal will appear with an overview of your new mock server's details. Note the mock server's URL because you'll need it to make requets that return mock data. Learn more about [examples and servers](/docs/postman/collections/examples/).
```
Comment (claireriley) [inline src/pages/docs/postman/mock-servers/setting-up-mock.md:169]: "requets"

Comment (claireriley) [inline src/pages/docs/postman/mock-servers/setting-up-mock.md:169]: I wouldn't include this last sentence, seems out of place

File: src/pages/docs/postman/mock-servers/setting-up-mock.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (JJGothard) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md:49]: Done

Comment (JJGothard) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md]: Done

File: src/pages/docs/postman/mock-servers/setting-up-mock.md [hunk 15]
```diff
@@ -128,7 +128,13 @@ If you already have mock servers created from the same collection, you will see
 
 ![Create new mock server from collection sidebar when there are existing mock servers](https://assets.postman.com/postman-docs/mock-cnx-collection-sidebar-existing.png)
```
[the remark this answers is not in the document]
Comment (JJGothard) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md]: Done

File: src/pages/docs/postman/mock-servers/setting-up-mock.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (JJGothard) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md]: Done

File: src/pages/docs/postman/mock-servers/setting-up-mock.md (hunk 11 above)
Comment (JJGothard) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md]: Done

File: src/pages/docs/postman/mock-servers/setting-up-mock.md (hunk 12 above)
Comment (JJGothard) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md]: Done

File: src/pages/docs/postman/mock-servers/setting-up-mock.md (hunk 13 above)
Comment (JJGothard) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md]: Done

File: src/pages/docs/postman/mock-servers/setting-up-mock.md (hunk 14 above)
Comment (JJGothard) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md]: Done

Comment (JJGothard) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md]: Done

**Comment by JJGothard** [comment]
@claireriley , all of your comments have been addressed, thanks!

File: src/pages/docs/postman/mock-servers/setting-up-mock.md [hunk 16]
```diff
@@ -126,9 +128,15 @@ This will launch the **Create mock server** modal. Follow the process [outlined
 
 If you already have mock servers created from the same collection, you will see an **Add mock** button instead of **Create a mock server**.
 
-![Create new mock server from collection sidebar when there are existing mock servers](https://assets.postman.com/postman-docs/mock-cnx-collection-sidebar-existing.png)
+![Create new mock server from collection sidebar when there are existing mock servers](https://assets.postman.com/postman-docs/Adding+another+mock.jpg)
```
Comment (claireriley) [inline src/pages/docs/postman/mock-servers/setting-up-mock.md:131]: Since this image is just a slightly different crop of https://assets.postman.com/postman-docs/New+collection+with+new+mock.jpg below, let's use https://assets.postman.com/postman-docs/New+collection+with+new+mock.jpg here

File: src/pages/docs/postman/mock-servers/setting-up-mock.md [hunk 17]
```diff
@@ -138,7 +146,35 @@ This will open the **Set up the mock server** modal. Pass the values outlined in
 
 This process of creating a mock server will also create a collection with the same title as the mock server. This collection will have a copy of the same request that you chose from the **History** sidebar. The mock server created is tied to this new collection.
 
-![New collection created when creating mock server from history request](https://assets.postman.com/postman-docs/mock-cnx-history-request-created.png)
+![New collection created when creating mock server from history request](https://assets.postman.com/postman-docs/New+collection+with+new+mock.jpg)
```
Comment (claireriley) [inline src/pages/docs/postman/mock-servers/setting-up-mock.md:149]: can we make this an image of you selecting to make a mock from a history request?

File: src/pages/docs/postman/mock-servers/setting-up-mock.md [hunk 18]
```diff
@@ -138,7 +146,35 @@ This will open the **Set up the mock server** modal. Pass the values outlined in
```
Comment (claireriley) [inline src/pages/docs/postman/mock-servers/setting-up-mock.md:146]: is what I'm seeing for ln 143-145 in your version? because I can't select it here i.e.

> Create new mock server from history request
This will open the **Set up the mock server** modal. Pass the values outlined in the [New button section](#mocking-from-the-new-button). Click **Create Mock Server**.

if so let's remove the numbers from that image

File: src/pages/docs/postman/mock-servers/setting-up-mock.md [hunk 19]
```diff
@@ -138,7 +146,35 @@ This will open the **Set up the mock server** modal. Pass the values outlined in
...
+
+#### Mocking all requests made on a specific date
+
+You can create a mock server from **History** for all the requests you made on a specific date, creating a new collection at the same time.
+
+In the Postman app, click **History**.
+
+![History tab](https://assets.postman.com/postman-docs/History+Mock+creation.jpg)
+
+Next to the date you want to mock all the requests for, click the ellipsis **(...)** and then click **Mock Requests**.
```
Comment (claireriley) [inline src/pages/docs/postman/mock-servers/setting-up-mock.md:159]: > click the ellipsis **(...)** and then click **Mock Requests**.

->

> click the ellipsis **(...)** > **Mock Requests**.

File: src/pages/docs/postman/mock-servers/setting-up-mock.md [hunk 20]
```diff
@@ -138,7 +146,35 @@ This will open the **Set up the mock server** modal. Pass the values outlined in
...
+
+![Ellipsis and mock requests](https://assets.postman.com/postman-docs/No+arrows+ellipsis+and+mocks.jpg)
+
+The **Set up the mock server** modal will open.
+
+In the modal, give your server a name. You also have the option to connect an environment to the server, make the server private, and to save the server's URL as an environment variable. Click **Create mock server**.
```
Comment (claireriley) [inline src/pages/docs/postman/mock-servers/setting-up-mock.md:165]: Since the app appears to always say "mock server" instead of just "server", let's use that in all mentions here

File: src/pages/docs/postman/mock-servers/setting-up-mock.md [hunk 21]
```diff
@@ -138,7 +146,35 @@ This will open the **Set up the mock server** modal. Pass the values outlined in
...
+
+![Mocks modal](https://assets.postman.com/postman-docs/Second+step+modal.jpg)
+
+> If you choose to make your mock server private, you'll have to generate an API key.
```
Comment (claireriley) [inline src/pages/docs/postman/mock-servers/setting-up-mock.md:169]: can you make "generate an API key" the same link that the app has?

File: src/pages/docs/postman/mock-servers/setting-up-mock.md [hunk 22]
```diff
@@ -138,7 +146,35 @@ This will open the **Set up the mock server** modal. Pass the values outlined in
...
+
+The "Next steps" modal will appear with an overview of your new mock server's details. Note the mock server's URL because you'll need it to make requests that return mock data.
+
+![Mocks modal](https://assets.postman.com/postman-docs/Next+steps+modal+RS.jpg)
+
+Creating the mock server will also create a collection with the same name as the server. The collection will have a copy of each request made on the date you chose in your **History**. All requests from this collection will be made to the server you just created.
+
+![Mocks modal](https://assets.postman.com/postman-docs/Collection+and+server+Mocks.jpg)
```
Comment (claireriley) [inline src/pages/docs/postman/mock-servers/setting-up-mock.md:177]: Is this showing the same info as https://assets.postman.com/postman-docs/New+collection+with+new+mock.jpg? If so, let's use that one for a fuller crop.

File: src/pages/docs/postman/mock-servers/setting-up-mock.md (hunk 16 above)
Comment (JJGothard) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md]: @claireriley , per @SueSmith 's comments on another PR, I'm going with a different, non-Covid-19 image. Done

File: src/pages/docs/postman/mock-servers/setting-up-mock.md (hunk 18 above)
Comment (JJGothard) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md:144]: @claireriley , I'm not sure if you mean whether or not the image is in my version of the doc or if the capability is in my version of the app. If the latter, yes, it is. Also, this content was pre-existing and outside the knowledge gap this PR was meant to fill per the gaps analysis sheet (single requests having been already documented by someone else, these image were not mine). I changed the structure to include the second subsection on mocking all of the requests made on a single date but did not change any of the content. I did change this screenshot just now. Done

File: src/pages/docs/postman/mock-servers/setting-up-mock.md (hunk 17 above)
Comment (JJGothard) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md:147]: @claireriley , this screenshot is meant to reflect the new mock and the new collection of the same name described in the paragraph, something that happens after the mock is created. I'm not sure how an image of creating a mock illustrates that. This is also content that was existing and outside the scope of the knowledge gap this PR is meant to address.

File: src/pages/docs/postman/mock-servers/setting-up-mock.md (hunk 19 above)
Comment (JJGothard) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md:157]: Done

File: src/pages/docs/postman/mock-servers/setting-up-mock.md (hunk 17 above)
Comment (JJGothard) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md]: I did change the image to not be related to Covid-19, per other reviews

File: src/pages/docs/postman/mock-servers/setting-up-mock.md (hunk 20 above)
Comment (JJGothard) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md]: Done

File: src/pages/docs/postman/mock-servers/setting-up-mock.md (hunk 21 above)
Comment (JJGothard) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md]: Done

File: src/pages/docs/postman/mock-servers/setting-up-mock.md (hunk 22 above)
Comment (JJGothard) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md]: I replaced that image because there was a disconnect between the name of the collection and the name of the server in it. I've done the same here.

File: src/pages/docs/postman/mock-servers/setting-up-mock.md (hunk 18 above)
Comment (claireriley) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md:144]: I tend to rewrite entire pages when I fill in gaps as I prefer to come back to them only with updates, but as long as reviewing the rest of the content is on your radar at some point that's fine. I only mentioned this bit as it was displaying oddly in github and I couldn't tell if it was staying, edited, or deleted.

**Comment by JJGothard** [comment]
@claireriley , all comments have been addressed here.

File: src/pages/docs/postman/mock-servers/setting-up-mock.md (hunk 18 above)
Comment (JJGothard) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md:144]: That makes sense, thanks. I think it might be a good idea for the three of us to talk about scope of these kinds of knowledge gap assignments so we're all on the same page. That could be especially helpful to me because I know that a lot of these docs have been recently updated, but I don't know which ones. Like those Documentation knowledge gap tasks that got reassigned to me from Raja: those pages, from what I've been told, have all been recently updated, so I would focus/prioritize just filling the gap. Does that makes sense?

Comment (claireriley) [reply src/pages/docs/postman/mock-servers/setting-up-mock.md:144]: Sure we can sync on this @SueSmith

**Comment by JJGothard** [comment]
@SueSmith , it looks like I'd addressed all of Claire's comments on 4/20, and that was the last action item. Thanks!

**Comment by SueSmith** [comment]
@JJGothard k as I mentioned on slack if you're waiting on someone else and have already tagged them on github if you just give them a follow-up ping in slack, it's easy to miss gh notifications.. 👍

**Comment by JJGothard** [comment]
@claireriley , I've addressed all of your comments here. Thanks! 😊

**Comment by SueSmith** [comment]
@JJGothard I take it you've also pinged in slack as per my comment above? ^^

**Comment by JJGothard** [comment]
Pinged in Slack: Check.  @SueSmith 👍

**Comment by SueSmith** [comment]
@JJGothard looks like some conflicts have arisen.. hopefully not too big a deal to sort 🤞

**Comment by JJGothard** [comment]
Merge conflicts resolved, @SueSmith 😁
